doc/building_blocks/index.rst

```diff
@@ -2,21 +2,17 @@
 
 .. _building_blocks:
 
-============================
-The nilearn building blocks
-============================
+================================================
+Advanced usage: manual pipelines and scaling up
+================================================
 
 In this section of the nilearn documentation, we describe the building
 blocks that can be assembled to create a machine-learning data analysis
 pipeline. This section is slightly more advanced and abstract than the
-previous section, on specific :ref:`data_analysis`. It provides knowledge
-that is useful to go off the beaten path in terms of data processing.
+other. It provides knowledge that is useful to go off the beaten path in
+terms of data processing.
 
 .. toctree::
 
-   pipeline_introduction
-   data_preparation
-   supervised_learning
-   unsupervised_learning
-   plotting.rst
-   manipulating_mr_images.rst
+   manual_pipeline.rst
```

Comment:
There are a number of files deleted without replacement.  Just wanted to check that this is intended (rather than leaving those files there until the documentation is improved).